quinn/Cargo.toml

```diff
@@ -45,7 +45,7 @@ socket2 = { workspace = true }
 thiserror = { workspace = true }
 tracing =  { workspace = true }
 tokio = { workspace = true }
-udp = { package = "quinn-udp", path = "../quinn-udp", version = "0.5", default-features = false }
+udp = { package = "quinn-udp", path = "../quinn-udp", version = "0.5", default-features = false, features = ["tracing"] }
```

Comment:
As mentioned previously, this should be `tracing-log`.

Reply:
Setting `features = ["tracing-log"]` would effectively make `tracing/log` non-optional. Am I missing something?

Reply:
I guess we should not (unconditionally) enable `tracing` here in general, it is enabled by default via the quinn `default` features anyway as before.